src/sage.c

```diff
@@ -270,6 +270,10 @@ int32_t sage_per_forest(const int forestnr, struct save_info *save_info,
     }
 
 #else
+    /*MS: This is the normal SAGE processing on a tree-by-tree (vertical) basis */
+    
+    /* Now start the processing */
+    int galaxycounter = 0;
```

Comment:
This one is important to specify that it's `_32t` I think.

Reply:
I have changed this one